tests/test-dirs/expand_node/ppx-tests.t/rewriter/dune

```diff
@@ -0,0 +1,4 @@
+(library
+ (name my_ppx)
+ (kind ppx_rewriter)
+ (libraries ppxlib))
```

Comment:
Be careful to always have a newline at the end of the files. It is an old tradition and that's the reason why github is displaying this ugly ⛔️ sign. You can configure your editor to always add a last line.

(also happens in other files :-) )